http_stubs/models.py

```diff
@@ -108,6 +108,7 @@ class LogEntry(models.Model):
 
     path = models.URLField(
         verbose_name='Full request path',
+        max_length=2000,
```

Comment:
Migration is required here, because URLField is CharField, so changes won't be effective without changing size of the field in the database.